openedx/core/djangoapps/course_groups/cohorts.py

```diff
@@ -452,7 +481,7 @@ def set_course_cohort_settings(course_key, **kwargs):
         A CourseCohortSettings object.
 
     Raises:
-        ValueError if course_key is invalid.
+        Http404 if course_key is invalid.
```

Comment:
did exception handling actually change in this PR or is this just documentation being corrected?